### DISCUSSION TITLE
osu!lazer score system

### DISCUSSION THREAD

**Discussion by kitoraosu** [ppy/osu#25922] [Ideas]
(((I use Translator)))
![image](https://github.com/ppy/osu-web/assets/30956599/a37a23d5-c7fa-4544-94a5-ac59defaee9d)
Previously, **osu!laser** had the ability to set the old _score system (**from stable**)_ through the settings.
Settings now:
----Classic
----Standardized
Do not change the scoring system _(always **score v2** (or other))_
--_Points are taken (minus) away when you miss_

It seems to me that many players will find it more convenient to look at the old scoring system, just like me

**Comment by peppy** [comment]
I'm having trouble understanding your point here. Are you saying that classic should be the default?

**Reply by bdach** [reply]
it's either that or OP doesn't like that classic scoring is basically a ballparked estimate and has the same behaviour as score V2 in that missing takes away score (which it would never do if classic was "proper" score v1, but we obviously cannot implement that).
